### ISSUE TITLE
Record File Download Name Instead of ID

### ISSUE THREAD

**Issue by spencerfinnell** [easydigitaldownloads/easy-digital-downloads#6623]
### Expected behavior

When I view the file download log I expect to see the name of the actual file that was downloaded.

### Actual behavior

I see the current download file name.

### Steps to reproduce the behavior
1) Create a download with 1 file.
2) Buy download and download file.
3) Replace 1 file with another.
4) Download file.
5) Check logs.

### Justification or use case

Selling software you want the latest version of the file to be what gets downloaded. So you replace the single file attached to the download and update the file name with the latest version. The File ID that gets logged always remains the same so you cannot determine which versions of the files are being downloaded.

https://github.com/easydigitaldownloads/easy-digital-downloads/blob/master/includes/download-functions.php#L657

**Comment by JJJ** [comment]
@spencerfinnell would it be acceptable to keep the file ID and also include the file name as a new parameter?

**Comment by spencerfinnell** [comment]
Certainly!

**Comment by spencerfinnell** [comment]
You're right both should definitely be recorded. Updated the title to reflect.
